datadog_checks_dev/datadog_checks/dev/utils.py

```diff
@@ -182,6 +183,24 @@ def get_here():
     return get_parent_dir(inspect.currentframe().f_back.f_code.co_filename)
 
 
+def load_jmx_config():
+    root = get_parent_dir(inspect.currentframe().f_back.f_code.co_filename)
+    while True:
+        if file_exists(path_join(root, 'setup.py')):
+            break
+
+        new_root = os.path.dirname(root)
+        if new_root == root:
+            return {}
```

Comment:
What do you think about raising an exception here? I think if you use this function and there is no metrics.yaml, that's a bug.